src/SessionInstance.php

```diff
@@ -83,19 +83,21 @@ private function init()
         }
         $this->init = true;
 
-        session_cache_limiter(false);
+        if ( session_status() !== PHP_SESSION_ACTIVE ) {
+            session_cache_limiter(false);
 
-        session_set_cookie_params($this->cookie->getLifetime(), $this->cookie->getPath(), $this->cookie->getDomain(), $this->cookie->isSecure(), $this->cookie->isHttpOnly());
+            session_set_cookie_params($this->cookie->getLifetime(), $this->cookie->getPath(), $this->cookie->getDomain(), $this->cookie->isSecure(), $this->cookie->isHttpOnly());
```

Comment:
Hmm I'm not sure about this, as it's a bit misleading to create a session instance with custom cookie settings, but then have them quietly discarded.

Reply:
Would throwing an exception help your use case? That makes the most sense to me if you're happy with that.

Reply:
Yes, I think it would.  I'll update the PR shortly.